content/influxdb/v2.0/reference/upgrading/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Upgrade InfluxDB
+description: >
+  Upgrade your version of InfluxDB
+menu: influxdb_2_0_ref
+weight: 20
+influxdb/v2.0/tags: []
+---
+
```

Comment:
Let's update this reference on the Get started page:
To upgrade from InfluxDB 2.0 beta 16 or earlier to the latest version, see Upgrade to InfluxDB OSS 2.0rc0.
>>
To upgrade to InfluxDB 2.0rc1 (from InfluxDB 1.x or 2.0 beta 16 or earlier), see [Upgrade to InfluxDB](/influxdb/v2.0/reference/upgrading/).